<!DOCTYPE html><html><head><title>Ordinary</title><meta charSet="utf-8"/><meta name="viewport" content="width=device-width, initial-scale=1"/><meta name="description" content="Blog"/><link rel="icon" href="/favicon.ico"/><meta name="next-head-count" content="5"/><link rel="preload" href="/_next/static/css/8818eef5b782c46b.css" as="style"/><link rel="stylesheet" href="/_next/static/css/8818eef5b782c46b.css" data-n-g=""/><link rel="preload" href="/_next/static/css/fe65cfa4c7637815.css" as="style"/><link rel="stylesheet" href="/_next/static/css/fe65cfa4c7637815.css" data-n-p=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/_next/static/chunks/polyfills-5cd94c89d3acac5f.js"></script><script src="/_next/static/chunks/webpack-fd1bc4a65a80e5c8.js" defer=""></script><script src="/_next/static/chunks/framework-5f4595e5518b5600.js" defer=""></script><script src="/_next/static/chunks/main-a054bbf31fb90f6a.js" defer=""></script><script src="/_next/static/chunks/pages/_app-10d16e51767f2eb7.js" defer=""></script><script src="/_next/static/chunks/132-c8ad25a4bdd62a00.js" defer=""></script><script src="/_next/static/chunks/pages/posts/%5Bid%5D-fb712427f639438f.js" defer=""></script><script src="/_next/static/cItFC1ESUz_5ci48vv2CW/_buildManifest.js" defer=""></script><script src="/_next/static/cItFC1ESUz_5ci48vv2CW/_ssgManifest.js" defer=""></script><script src="/_next/static/cItFC1ESUz_5ci48vv2CW/_middlewareManifest.js" defer=""></script></head><body><div id="__next" data-reactroot=""><div class="Home_container__97eC3"><div class="Header_header__ubBbX"><div class="Header_headerLeft__gJBl3"><a href="/">Ordinary</a></div><div class="Header_headerRight__uflv6"><a href="/about">About</a></div></div><article class="Post_blogContent__h2Uga"><div><h1>invalid_token_response</h1>ordilov /<!-- --> <!-- -->2/8/202<h2>[invalid_token_response]</h2>
<p><code>An error occurred while attempting to retrieve the OAuth 2.0 Access Token Response</code>
직전까지 잘되던 OAuth2 인증에서 에러가 발생했습니다.
서버에 에러 로그도 없이 클라이언트에서 보이는 에러였기에 문제점을 찾지 못했습니다.
그러다가 어쩌다가 OAUth2 Client Secret Key를 보니 이전에 설정했던 것과 달라져 있었습니다.
주기적으로 테스트용 구글 client secret key가 만료가 되어 바뀌는 것이 문제점이었습니다.</p></div></article></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"metadata":{"layout":"post","title":"invalid_token_response","date":"2022-02-09 00:01:00 +0900","category":"trouble","tags":["trouble"],"keywords":["trouble"],"id":"invalid-token"},"mdxSource":{"compiledSource":"/*@jsxRuntime automatic @jsxImportSource react*/\nconst {Fragment: _Fragment, jsx: _jsx, jsxs: _jsxs} = arguments[0];\nconst {useMDXComponents: _provideComponents} = arguments[0];\nfunction MDXContent(props = {}) {\n  const {wrapper: MDXLayout} = Object.assign({}, _provideComponents(), props.components);\n  return MDXLayout ? _jsx(MDXLayout, Object.assign({}, props, {\n    children: _jsx(_createMdxContent, {})\n  })) : _createMdxContent();\n  function _createMdxContent() {\n    const _components = Object.assign({\n      h2: \"h2\",\n      p: \"p\",\n      code: \"code\"\n    }, _provideComponents(), props.components);\n    return _jsxs(_Fragment, {\n      children: [_jsx(_components.h2, {\n        children: \"[invalid_token_response]\"\n      }), \"\\n\", _jsxs(_components.p, {\n        children: [_jsx(_components.code, {\n          children: \"An error occurred while attempting to retrieve the OAuth 2.0 Access Token Response\"\n        }), \"\\n직전까지 잘되던 OAuth2 인증에서 에러가 발생했습니다.\\n서버에 에러 로그도 없이 클라이언트에서 보이는 에러였기에 문제점을 찾지 못했습니다.\\n그러다가 어쩌다가 OAUth2 Client Secret Key를 보니 이전에 설정했던 것과 달라져 있었습니다.\\n주기적으로 테스트용 구글 client secret key가 만료가 되어 바뀌는 것이 문제점이었습니다.\"]\n      })]\n    });\n  }\n}\nreturn {\n  default: MDXContent\n};\n","frontmatter":{},"scope":{}}},"__N_SSG":true},"page":"/posts/[id]","query":{"id":"invalid-token"},"buildId":"cItFC1ESUz_5ci48vv2CW","isFallback":false,"gsp":true,"scriptLoader":[]}</script></body></html>